fe/fe-core/src/main/java/org/apache/doris/nereids/stats/JoinEstimation.java

```diff
@@ -39,9 +39,9 @@ private static Statistics estimateInnerJoin(Statistics crossJoinStats, List<Expr
                 .sorted((a, b) -> {
                     double sub = a.second - b.second;
                     if (sub > 0) {
-                        return -1;
-                    } else if (sub < 0) {
                         return 1;
+                    } else if (sub < 0) {
+                        return -1;
```

Comment:
why changed this

Reply:
because this formula
```
for (int i = 0; i < sortedJoinConditions.size(); i++) {
            sel *= Math.pow(sortedJoinConditions.get(i).second, 1 / Math.pow(2, i));
}
```
sel=sel0 * sel1^(1/2)+...
if sel0 > sel1, we have sel>sel0, but sel>sel1 is not always hold.

suppose we have filter `(a=b) and (c=d)`
sel1: selectivity of (a=b), sel1=0.5
sel2: selectivity of (c=d), sel2=0.04

sel= 0.5 * 0.2 = 0.25
then sel > sel2. this is wrong